### DISCUSSION TITLE
How set pvp disable in wilderness?

### DISCUSSION THREAD

**Discussion by Ecilpse** [TownyAdvanced/Towny#4986] [Q&A] [answered]
version: 0.97.0.0
paper 1.16.5

sorry for my poor english.
i want set PvP disable in wilderness.
my server was installed multiverse, worldguard.
but when i set pvp deny through this plugin, pvp deny flags also overwrite town, plot pvp flags.
i want pvp disable only wilderness.
i also find towny config, database (Towny/data/worlds), but can't find wilderness PvP flags.

**Comment by LlmDl** [comment, marked as answer]
Use `/tw toggle pvp` until you see it is disabled.
